annotator/static/views/player.js

```diff
@@ -46,6 +46,12 @@ class PlayerView {
         // Video URL
         this.videoSrc = videoSrc;
 
+        // Video start time
+        this.videoStart = parseFloat(videoStart);
```

Comment:
Oh wait I forgot JSON is literally JavaScript

What you should do is `json.dump`/(aka `JSON.stringify`) it in Python so that the frontend can just use the variable as is.

1. query string is a string, goes to backend
2. backend parses it into float
3. backend uses json.dumps to convert float to JavaScript notation
4. frontend can use the value as is